apps/accounts/forms.py

```diff
@@ -357,6 +354,17 @@ class Meta:
 AsnFormset = forms.formset_factory(AsnForm, formset=MoreConvenientFormset, extra=0)
 
 
+class ExtraNetworkForm(forms.Form):
```

Comment:
It just occurred to me that at this point we're accepting the text input in the newly added form (explanation), but do not persist it. Which means it will not be accessible by anyone after the form is submitted.

If you want to make the explanation available to GWF staff, you should add some logic in the view to persist this data

Reply:
Duh! I'll add that extra line and migration file now